reactDashboard/src/components/HeatMap/Heatmap.css

```diff
@@ -78,21 +98,60 @@
 .heatmapScale .l-6 {
   background-color: #bd0026;
 }
+
+.darkmode .heatmapScale .l-0,
+.darkmode .heatmapScale .l-1,
+.darkmode .heatmapScale .l-2,
+.darkmode .heatmapScale .l-3 {
+  color: #020202;
+}
+
+/* to meet accessibility requirements */
+.heatmapScale .l-6,
+.heatmapScale .l-7 {
+  color: white;
+}
+
 .heatmapScale .l-7 {
   background-color: #020202;
+}
+
+.darkmode .heatmapScale .l-7 {
   color: white;
 }
 
 .heatmapScale {
-  padding-top: 5px;
-  padding-bottom: 5px;
+  text-align: center;
+  padding-top: 0;
+  padding-bottom: 0;
+  margin-bottom: 10px;
+  margin-left: 0;
+  border: 1px solid darkgrey;
+  width: 100%;
+  display: flex;
+  flex-wrap: nowrap;
+  margin-right: 0;
+}
+
+.darkmode .heatmapScale {
+  border-color: #f2f2f2;
+  background-color: #f2f2f2;
+  color: #020202;
 }
+
 .heatmapScale span {
-  padding: 5px 10px;
+  padding: 2px 15px;
 }
+
 .heatmapScale span.smallscale {
   display: none;
 }
+
+.heatmapScale span {
```

Comment:
This block can be moved up to replace the one at line 142

Reply:
refactored